api/app/signals/apps/signals/migrations/0129_new_subcategory_kerstbomen.py

```diff
@@ -0,0 +1,66 @@
+"""
+Category changes SIG-3347
+
+New subcategory for Kerstbomen (christmas trees)
+"""
+from django.db import migrations
+
+# SIG-3347
+NEW_CATEGORIES = {
+    'afval': {
+        'kerstbomen': {
+            'name': 'Kerstbomen',
+            'description': 'Meldingen over aangeboden/gedumpte Kerstbomen',
+            'handling': 'A3DMC',
+            'handling_message':
+                'Wij hebben uw melding doorgestuurd naar de uitvoering. '
+                'Normaal gesproken handelen wij uw melding binnen 3 werkdagen af. '
+                'Helaas is in verband met corona ons ziekteverzuim hoger dan normaal. '
+                'Wij doen ons uiterste best om uw melding zo snel mogelijk af te handelen. '
+                'Uiteraard houden wij rekening met meldingen die urgent zijn.',
+            'departments': ['AEG'],
+            'slo': '3W',
+        }
+    }
+}
+
+
+def _new_categories(apps, schema_editor):
+    Category = apps.get_model('signals', 'Category')
+    Department = apps.get_model('signals', 'Department')
+    ServiceLevelObjective = apps.get_model('signals', 'ServiceLevelObjective')
+
+    for main_category_slug, data in NEW_CATEGORIES.items():
+        main_category = Category.objects.get(slug=main_category_slug, parent__isnull=True)
+
+        for category_slug, category_data in data.items():
+            category = Category.objects.create(name=category_slug, parent=main_category)  # noqa Using the slug as name to ensure the slug is correctly created
+
+            category.name = category_data['name']
+            category.description = category_data['description']
+            category.handling = category_data['handling']
+            category.handling_message = category_data['handling_message']
+
+            responsible_deps = Department.objects.filter(code__in=category_data['departments'])
+            category.departments.add(*responsible_deps, through_defaults={'is_responsible': True, 'can_view': True})
+            # all departments have visibility on these categories, hence:
```

Comment:
This is because of the "bijzondere rechten op deze subcategorie voor andere SIA gebruikers: Geen"?

Reply:
Yes, can be changed through the FE if needed.